schema/iguana.owx

```diff
@@ -0,0 +1,1395 @@
+<?xml version="1.0"?>
+<Ontology xmlns="http://www.w3.org/2002/07/owl#"
+     xml:base="https://iguana-benchmark.eu/ontology/"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     ontologyIRI="https://iguana-benchmark.eu/ontology/"
+     versionIRI="https://iguana-benchmark.eu/ontology/4.0.0/">
+    <Prefix name="dc" IRI="http://purl.org/dc/elements/1.1/"/>
+    <Prefix name="owl" IRI="http://www.w3.org/2002/07/owl#"/>
+    <Prefix name="rdf" IRI="http://www.w3.org/1999/02/22-rdf-syntax-ns#"/>
+    <Prefix name="xml" IRI="http://www.w3.org/XML/1998/namespace"/>
+    <Prefix name="xsd" IRI="http://www.w3.org/2001/XMLSchema#"/>
+    <Prefix name="iont" IRI="https://iguana-benchmark.eu/ontology/class/"/>
+    <Prefix name="ires" IRI="https://iguana-benchmark.eu/ontology/resource/"/>
+    <Prefix name="rdfs" IRI="http://www.w3.org/2000/01/rdf-schema#"/>
+    <Prefix name="skos" IRI="http://www.w3.org/2004/02/skos/core#"/>
+    <Prefix name="iprop" IRI="https://iguana-benchmark.eu/ontology/properties/"/>
+    <Prefix name="schor" IRI="https://w3id.org/scholarlydata/role/"/>
+    <Prefix name="dcterms" IRI="http://purl.org/dc/terms/"/>
```

Comment:
Did you add the prefixes or was that Protege?

Some of them are never used, i.e. schor and skos

Reply:
They were already added before I changed the ontology so I just kept them. I don't know who/what added them.
Should I remove them?

Reply:
I removed them.